internal/backend/dashboardsvc/vars.go

```diff
@@ -84,9 +85,15 @@ func (s Svc) genVarsList(w http.ResponseWriter, r *http.Request, sid sdktypes.Va
 			if cv.IsSecret() {
 				v = "🤐"
 			}
+
+			n := cv.Name().String()
+			if cv.IsRequired() {
+				n = "<em>" + n + "</em>"
```

Comment:
This seems hacky, should be a template for the vars and inside the template you should check `IsRequired`

Reply:
of course it is hacky, it's the ugly dashboard.